modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/js/PageTypeSelector.es.js

```diff
@@ -86,17 +92,37 @@ function PageTypeSelector(props) {
 								{Liferay.Language.get('add-page')}
 							</ClayDropDown.Item>
 						)}
-						<ClayDropDown.Item
-							data-value={Liferay.Language.get('configure')}
-							key={Liferay.Language.get('configure')}
-							onClick={handleOnConfigureClick}
-							title={Liferay.Language.get('configure')}
-						>
-							{Liferay.Language.get('configure')}
-						</ClayDropDown.Item>
+						{props.addCollectionLayoutURL !== '' && (
```

Comment:
If we want to cover undefined and null cases maybe using coercion makes it simpler:
`props.addCollectionLayoutURL`

Reply:
👍